### ISSUE TITLE
Add a preliminary visualization frontend 

### ISSUE THREAD

**Issue by bisejdiu** [ProLint/ufcc#12]
Not clear yet how we should go around implementing this. The actual features to implement are even less clear. 
We can start with a backbone setup and continue from there.

**Comment by bisejdiu** [comment]
Development is ongoing, and can, for the time being, be followed [here](/ProLint/ufcc/tree/server).

**Comment by bisejdiu** [comment]
The dashboard is very close to being usable. A few more improvements are needed, but the ProLint dashboard is close to being done. 

- We need to decide if a 3D viewer can be supported (at least as part of the dashboard)
- Test the dashboard with more diverse input systems. 
- Decide if a proper SQL backend is needed to temporarily (or permanently?) store data. 
- The server code needs to be improved in terms of readability and formatting.

**Comment by danielpastor97** [comment]
Done!

**Comment by bisejdiu** [comment]
No, not yet. There are still more things we need to do (not to mention bug fixes). 
I just pushed a new branch that adds a "shared contacts" network application to the dashboard.
